README.md

```diff
@@ -1,3 +1,38 @@
 # btc-transaction-utils
 
-TODO
\ No newline at end of file
+[![Travis Build Status](https://img.shields.io/travis/exonum/btc-transaction-utils/master.svg?label=Linux)](https://travis-ci.org/exonum/btc-transaction-utils)
+[![Appveyor Build Status](https://img.shields.io/appveyor/ci/exonum-org/btc-transaction-utils/master.svg?label=Windows)](https://ci.appveyor.com/project/exonum-org/btc-transaction-utils)
+[![dependency status](https://deps.rs/repo/github/exonum/btc-transaction-utils/status.svg)](https://deps.rs/repo/github/exonum/btc-transaction-utils)
+[![Docs.rs](https://docs.rs/btc-transaction-utils/badge.svg)](https://docs.rs/btc-transaction-utils)
+![rust 1.23+ required](https://img.shields.io/badge/rust-1.23+-blue.svg?label=Required%20Rust)
+
+BTC transaction utils is a small library that helps to create multisig addresses
+and to swiftly sign some types of segwit transactions as well as to check the
+existing signatures, if required.
+
+Manipulations with segwit transactions are quite intricated, so the main goal
```

Comment:
"Complicated" fits well enough and doesn't have to be added to the `cspell` dictionary, doesn't it?